src/Groups/GAPGroupsLibrary.jl

```diff
@@ -0,0 +1,476 @@
+export
+    all_primitive_groups,
+    all_small_groups,
+    all_transitive_groups,
+    blocks,
+    isperfect,
+    isprimitive,
+    isregular,
+    issemiregular,
+    istransitive,
+    maximal_blocks,
+    moved_points,
+    number_moved_points,
```

Comment:
It doesn't really make sense to have these functions in `GAPGroupsLibrary.jl`. Most of them are relevant to all G-Sets (see issue #108), so perhaps move them to a new file `gsets.jl` (also `representatives_minimal_blocks`, `transitivity`; but not `isperfect`, that should go to wherever we have methods like `isabelian`, `isnilpotent` etc. -- so perhaps just move it back to `GAPGroups.jl`?

Reply:
I just moved these methods into this file because it looks more coherent to have `isprimitive` together with the other functions handling primitive groups, and same the others. But I can move it back in `GAPGroups.jl` if you think it's better.

Reply:
Yes I think it is better! I understand your argument, but really, this relation of the functions is superficial; properties like `isperfect` are relevant for all groups, not just those from a library